src/org/rascalmpl/values/RascalValueFactory.java

```diff
@@ -1369,6 +1393,11 @@ public CharWithKeywordParametersFacade(IConstructor content, io.usethesource.cap
             super(content, parameters);
         }
 
+		@Override
+		public int getConcreteMatchFingerprint() {
+			return ((ITree) content).getConcreteMatchFingerprint();
```

Comment:
codecov is saying these parse tree fingerprints aren't hit, is it possible to extend the tests for these?

Reply:
It would not help in finding bugs because this method simply forwards to the actual implementation, and without this method you'd get a Java compiler message saying that a method implementation is missing from the ITree interface. 

For good measure I can add an assert to every test for the annotated ITree case.